0006-cmx2cdp-protocol/README.md

```diff
@@ -1,84 +1,102 @@
 ---
 name: CMX to CDP protocol
 shortname: 6/CMX2CDP
-status: raw
-editor: Andrew Pasika <apasika@cybervisiontech.com>
+status: draft
```

Comment:
I asked @akokhanovskyi, he said I can, so I did. If the RFC is raw, I should not proofread it at all, I suppose. Guys, please talk to each other and let me know.